nuxeo-distribution/nuxeo-war-tests/itests.xml

```diff
@@ -0,0 +1,72 @@
+<project name="integration-tests"
+         xmlns:artifact="urn:nuxeo-artifact">
+  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build"/>
+  <taskdef resource="org/nuxeo/build/artifact/antlib.xml"
+           uri="urn:nuxeo-artifact"/>
+  <taskdef resource="net/sf/antcontrib/antlib.xml"/>
+
+  <property name="out.dir" value="${maven.project.build.directory}"/>
+  <property name="nuxeo.pack" value="${out.dir}/nuxeo-war.zip"/>
+  <unzip dest="${out.dir}/" overwrite="false">
+    <artifact:resolveFile key="org.nuxeo:nuxeo-ftest::zip"/>
+  </unzip>
+  <import file="${out.dir}/nuxeo-ftest.xml"/>
+
+  <target name="prepare-environment"
+          depends="_init,prepare-db,prepare-tomcat,prepare-monitor"
+          description="prepare all ressources for test">
+
+    <antcall target="create-pack"/>
+    <antcall target="download-tomcat"/>
+    <antcall target="deploy-pack"/>
+
+    <antcall target="_use-catalina"/>
+  </target>
+
+  <target name="start" depends="_init,stop,start-unix,start-windows">
+    <!-- Sadly, Tomcat startup return too quickly -->
+    <sleep seconds="60"/>
+  </target>
+
+  <target name="stop" depends="_init,_use-catalina,stop-unix,stop-windows">
+    <if>
+      <equals arg1="${env.KILL_TOMCAT}" arg2="true"/>
+      <then>
+        <antcall target="kill-tomcat"/>
+      </then>
+    </if>
+  </target>
+
+  <target name="download-tomcat">
+    <property name="tomcat.gav" value="org.apache.tomcat:tomcat::zip"/>
+    <echo level="info" message="Downloading ${tomcat.gav}..."/>
+    <delete failonerror="false" dir="${nuxeo.home}"/>
+    <unzip dest="${nuxeo.home}/">
+      <artifact:file key="${tomcat.gav}"/>
+      <cutdirsmapper dirs="1"/>
+    </unzip>
+  </target>
+
+  <target name="create-pack">
+    <delete failonerror="false" file="${nuxeo.pack}"/>
+    <chmod dir="${nuxeo.home}/bin" perm="ug+x" includes="*.sh,*ctl"/>
+    <exec executable="${nuxeo.ctl}" spawn="false" failonerror="true">
+      <arg value="pack"/>
+      <arg value="-d"/>
+      <arg value="--"/>
+      <arg value="${nuxeo.pack}"/>
+    </exec>
+  </target>
+
+  <target name="deploy-pack">
+    <unzip src="${nuxeo.pack}" dest="${nuxeo.home}/"/>
+  </target>
+
+  <target name="_use-catalina">
+    <!-- Reset ${nuxeo.ctl} for using `catalina.sh` instead -->
+    <var name="nuxeo.ctl" unset="true"/>
+    <condition property="nuxeo.ctl" value="${nuxeo.home}/bin/catalina.sh" else="${nuxeo.home}/bin/catalina.bat">
+      <isset property="osfamily-unix"/>
+    </condition>
+  </target>
```

Comment:
why the duplication versus https://github.com/nuxeo/tools-nuxeo-ftest/pull/7/files#diff-36413b253468781197566da44d4ca81eR253 ?